libs/s25main/world/GameWorldView.cpp

```diff
@@ -95,7 +97,11 @@ void GameWorldView::SetZoomFactor(float zoomFactor, bool smoothTransition /* = t
     else
         targetZoomFactor_ = zoomFactor;
     if(!smoothTransition)
+    {
         zoomFactor_ = targetZoomFactor_;
+        UpdateEffectiveZoomFactor();
+        CalcFxLx();
```

Comment:
I asked on Discord whether this call to `CalcFxLx()` was missing. Added here because I think it is.

Reply:
Yeah, I guess we never used the non-smooth transition often and hence didn't notice.